proto/message.py

```diff
@@ -269,6 +287,14 @@ def __new__(mcls, name, bases, attrs):
     def __prepare__(mcls, name, bases, **kwargs):
         return collections.OrderedDict()
 
+    def add_to_class(cls, name, value):
```

Comment:
Note: This whole `contribute_to_class` pattern is heavily (or, more like entirely) [borrowed from Django](https://github.com/django/django/blob/a77c9a4229cfef790ec18001b2cd18bd9c4aedbc/django/db/models/base.py#L324-L328).